ibazel/fswatcher/fswatcher.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fswatcher
+
+import (
+	"github.com/bazelbuild/bazel-watcher/ibazel/fswatcher/common"
+)
+
+type Event = common.Event
+type Op = common.Op
+
+const Create = common.Create
```

Comment:
Why type alias things that are already in the repo rather than just directly refer to them?

Reply:
Yea, I think this was a refactoring artifact. Thanks for the callout.